### ISSUE TITLE
add plateau callback to keras

### ISSUE THREAD

**Issue by simonlousky** [AsherYartsevTech/Mask_RCNN#22]
decrease learning rate at accuracy plateau

**Comment by AsherYartsevTech** [comment]
callbacks added!
